tests/System/Controller/SecuredViewHandlerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+declare(strict_types=1);
+namespace PhpList\RestBundle\Tests\System\Controller;
```

Comment:
To keep things consistent, please let's have a blank line between the `declare` line and the `namespace` section.